test/map.rs

```diff
@@ -354,11 +366,12 @@ fn test_commute_quickcheck_bug() {
 #[test]
 fn test_idempotent_quickcheck_bug1() {
     let ops = vec![
-        map::Op::Up {
-            dot: Dot::new(21, 5),
-            key: 0,
-            op: map::Op::Nop,
-        },
+        // TODO: ???
+        // map::Op::Up {
+        //     dot: Dot::new(21, 5),
+        //     key: 0,
+        //     op: map::Op::Nop,
+        // },
```

Comment:
I don't get the purpose of this test, what would be the appropriate replacement for it?

Reply:
I wish I would have taken better notes here, this was a bug that came up through quickcheck fuzz testing.

This first Op with nested Nop would create an empty `Map` value under the `0` key and I think that was the crux of this bug, something about the empty Map broke down on merge.

I think the closest replacement would be to replace this Nop with 2 new ops
1. one that adds a value to a nested map under key 0, ie. `{0: {32: 0}}`
2. and a following op that removes the `32` key from the sub map, ie. `{0: {}}`